src/briefcase/platforms/linux/system.py

```diff
@@ -1162,6 +1197,7 @@ def _package_pkg(self, app: AppConfig, **kwargs):  # pragma: no-cover-if-is-wind
                     [
                         "makepkg",
                     ],
+                    env={"PKGEXT": ".pkg.tar.zst"},
```

Comment:
In one of the environments I tested, some other random compression was used. So, forcing `zstd` now.